File: src/main/java/com/skidders/sigma/events/impl/Render2DEvent.java
package com.skidders.sigma.events.impl;

import com.skidders.sigma.events.Event;
import lombok.RequiredArgsConstructor;
import net.minecraft.client.util.math.MatrixStack;

@RequiredArgsConstructor
public class Render2DEvent extends Event {
    public final MatrixStack matrixStack;
}

File: src/main/kotlin/com/skidders/sigma/events/impl/Render2DEvent.kt
package com.skidders.sigma.events.impl;

import com.skidders.sigma.events.Event;
import lombok.RequiredArgsConstructor;
import net.minecraft.client.util.math.MatrixStack;

@RequiredArgsConstructor
data class Render2DEvent(val matrixStack: MatrixStack) : Event() {
}
